tpp-app/src/home/pages/CalendarScreen.js

```diff
@@ -19,10 +19,13 @@ const sideComponentWidth = 120
 
 
 // The component that is used by each day in the calendar
-const DayComponent = ({ date, state, marking, navigation }) => {
+const DayComponent = ({ date, state, marking, navigation, calendarData }) => {
+
+
+    
     return(
         <TouchableOpacity onPress={() => navigation.navigate(STACK_SCREENS.LOG_SYMPTOMS, {"date": date})}>
-            <View style={styles.dayContainer}>
+            <View style={{...styles.dayContainer}}>
```

Comment:
isn't this equivalent to just writing `style={styles.dayContainer}`?

Reply:
oh yea, fixed it!